frontend/src/routes/index.svelte

```diff
@@ -85,6 +86,14 @@
     if (!$filters.showNoProviders) {
       query += `&only_providers=true`
     }
+
+    if (!input) {
+      if ($sorting.byPopularity) {
+        query += `&popularity=${$sorting.asc ? "asc" : "desc"}`
+      } else if ($sorting.byReleaseDate) {
+        query += `&release_date=${$sorting.asc ? "asc" : "desc"}`
+      }
+    }
```

Comment:
Can we have a third option(byDefault(?)) that does this with the !input and have the old behavior on byPopularity and byReleaseDate?